rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/topic_cache.hpp

```diff
@@ -21,6 +21,7 @@
 #include <set>
 #include <sstream>
 #include <string>
+#include <tuple>
```

Comment:
nitpick: Is this actually used in this file?

Reply:
Removed.